packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -55,28 +55,27 @@ exports[`generate graphqlSchemaFromRDSSchema creates graphql schema from "postgr
 
 
 type Article @refersTo(name: \\"articles\\") @model {
-  pubId: Int! @refersTo(name: \\"pub_id\\") @default(value: \\"NULL\\") @primaryKey(sortKeyFields: [\\"pubType\\"])
+  pubId: Int! @refersTo(name: \\"pub_id\\") @primaryKey(sortKeyFields: [\\"pubType\\"])
   pubType: String! @refersTo(name: \\"pub_type\\") @default(value: \\"'A'::bpchar\\")
-  placeholder: String @default(value: \\"NULL\\")
+  placeholder: String!
 }
 
 type Comment @refersTo(name: \\"comments\\") @model {
-  pubId: Int! @refersTo(name: \\"pub_id\\") @default(value: \\"NULL\\") @primaryKey(sortKeyFields: [\\"commentTimestamp\\", \\"commenterEmail\\"])
+  pubId: Int! @refersTo(name: \\"pub_id\\") @primaryKey(sortKeyFields: [\\"commentTimestamp\\", \\"commenterEmail\\"])
   commentTimestamp: AWSDateTime! @refersTo(name: \\"comment_timestamp\\")
-  commenterEmail: String! @refersTo(name: \\"commenter_email\\") @default(value: \\"NULL\\")
-  commentText: String @refersTo(name: \\"comment_text\\") @default(value: \\"NULL\\")
+  commenterEmail: String! @refersTo(name: \\"commenter_email\\")
 }
 
 type Publication @refersTo(name: \\"publications\\") @model {
-  pubId: Int! @refersTo(name: \\"pub_id\\") @default(value: \\"NULL\\") @primaryKey @index(name: \\"publications_superkey\\", sortKeyFields: [\\"pubType\\"])
-  pubType: String @refersTo(name: \\"pub_type\\") @default(value: \\"NULL\\")
-  pubUrl: String @refersTo(name: \\"pub_url\\") @default(value: \\"NULL\\") @index(name: \\"publications_pub_url_key\\")
+  pubId: Int! @refersTo(name: \\"pub_id\\") @primaryKey @index(name: \\"publications_superkey\\", sortKeyFields: [\\"pubType\\"])
+  pubType: String @refersTo(name: \\"pub_type\\")
+  pubUrl: String! @refersTo(name: \\"pub_url\\") @index(name: \\"publications_pub_url_key\\")
 }
 
 type Story @refersTo(name: \\"stories\\") @model {
-  pubId: Int! @refersTo(name: \\"pub_id\\") @default(value: \\"NULL\\") @primaryKey(sortKeyFields: [\\"pubType\\"])
+  pubId: Int! @refersTo(name: \\"pub_id\\") @primaryKey(sortKeyFields: [\\"pubType\\"])
   pubType: String! @refersTo(name: \\"pub_type\\") @default(value: \\"'S'::bpchar\\")
-  placeholder: String @default(value: \\"NULL\\")
+  placeholder: String!
```

Comment:
These are expected as per the test schema definitions in https://github.com/aws-amplify/amplify-category-api/blob/palpatim.chore.merge-main-20231107/packages/amplify-graphql-schema-generator/src/__tests__/__utils__/schemas.ts